src/System.Diagnostics.DiagnosticSource/tests/System.Diagnostics.DiagnosticSource.Tests.builds

```diff
@@ -7,6 +7,10 @@
       <OSGroup>Windows_NT</OSGroup>
       <TestTFMs>netcore50;net46</TestTFMs>
     </Project>
+    <Project Include="System.Diagnostics.DiagnosticSource.Tests.csproj">
```

Comment:
The .builds files are no longer used. I suspect you didn't test any of these changes on desktop because I don't expect your desktop configuration even builds today.